tools/tools_common.h

```diff
@@ -154,6 +156,9 @@ parseArgs(int argc, char **argv) {
         case 'D':
             params.devicekey = optarg;
             break;
+        case 'f':
```

Comment:
Missing "f" in getopt_long. (--persist is recognized, -f is not)

Reply:
done